src/Options.ts

```diff
@@ -11,6 +11,11 @@ export interface IProps {
    */
   height?: number;
 
+  /**
+   * A number that will enable the snapping feature and create an intermediate point before opening the modal to full screen.
```

Comment:
Change the description according to the behavior of the props

Reply:
lol forgot to change it